frontend/src/main/scala/akkaviz/frontend/FrontendApp.scala

```diff
@@ -203,30 +205,27 @@ object FrontendApp extends JSApp with Persistence with PrettyJson with Manipulat
 
     repo.newActors.foreach {
       newActors =>
-        hierarchyView.insert(newActors)
+        hierarchyPanel.insert(newActors)
     }
 
+    setupComponents()
+  }
+
+  def setupComponents(): Unit = {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Use active names for operations with side effects](https://www.codacy.com/app/lustefaniak/akka-viz/file/2349669505/issues/source?bid=3121282&fileBranchId=3217398#l214)